jetson/odrive_bridge/src/__main__.py

```diff
@@ -220,21 +220,21 @@ def connect(self):
         modrive.set_current_lim(modrive.CURRENT_LIM)
         usb_lock.release()
 
-    def on_event(self, event):
+    def bridge_on_event(self, odrive_event):
         """
         Incoming events are
         delegated to the given states which then handle the event.
         The result is then assigned as the new state.
         The events we can send are disarm cmd, arm cmd, and calibrating cmd.
         """
 
-        print("on event called, event:", event)
+        print("on event called, event:", odrive_event)
 
-        self.state = self.state.on_event(event)
-        publish_state_msg(state_msg, odrive_bridge.get_state())
+        self.state = self.state.state_on_event(odrive_event)
+        publish_state_msg(state_msg, odrive_bridge.get_state_string())

```

Comment:
is there a separate function that just returns the state instead of the string?

Reply:
Not really but when I first saw the function, I assumed it was returning an enum and not a string so i was a little lost.  i can change it back

Reply:
ok nw can leave as is